requirements.txt

```diff
@@ -1,4 +1,4 @@
-disnake==2.3.0
+disnake==2.6.0
```

Comment:
something should probably be added to the contributing guide about requiring message content intent now